Files/Constants.cs

```diff
@@ -53,6 +53,23 @@ public static class PreviewPane
             /// The maximum number of pages loaded into the PDF preview.
             /// </summary>
             public const int PDFPageLimit = 50;
+            /// <summary>
+            /// The maximum file size, in bytes, that will attempted to be loaded as text if the extension is unkown.
```

Comment:
```suggestion
            /// The maximum file size, in bytes, that will attempted to be loaded as text if the extension is unknown.
```

Reply:
I need to get a spellchecker extension 😂